server/locks_controller.go

```diff
@@ -0,0 +1,143 @@
+package server
+
+import (
+	"bytes"
+	"fmt"
+	"html/template"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/gorilla/mux"
+	"github.com/pkg/errors"
+	"github.com/runatlantis/atlantis/server/events/locking"
+	"github.com/runatlantis/atlantis/server/events/models"
+	"github.com/runatlantis/atlantis/server/events/vcs"
+	"github.com/runatlantis/atlantis/server/logging"
+)
+
+// EventsController handles all webhook requests which signify 'events' in the
+// VCS host, ex. GitHub. It's split out from Server to make testing easier.
+type LockController struct {
+	AtlantisVersion    string
+	Locker             locking.Locker
+	Logger             *logging.SimpleLogger
+	VCSClient          vcs.ClientProxy
+	LockDetailTemplate TemplateWriter
+}
+
+var lockDeletedTemplate = template.Must(template.New("").Parse(
+	"The following Locks and plans deleted were deleted via the Atlantis UI:\n" +
```

Comment:
Right now it's only possible to delete a single lock right? So should this just be singular?

Reply:
Why is Locks capitalized?

Reply:
How about something like this:

![image](https://user-images.githubusercontent.com/1034429/38475116-71ead1e8-3b5a-11e8-957c-e64c8c58633f.png)

```
**Warning**: The plan for path: `.` workspace: `default` were deleted via the Atlantis UI.

To `apply` you must run `plan` again.
```